frontend/src/components/MainHeaderComponent.vue

```diff
@@ -107,7 +109,7 @@ const logoutBtnKind = computed(() =>
 
 const userHasAuthority = ["CLIENT_EDITOR", "CLIENT_ADMIN"].some((authority) =>
   ForestClientUserSession.authorities.includes(authority)
-);
+) && featureFlags.STAFF_SEARCH;
```

Comment:
This actually controls the visibility of the Create client button, instead of the Client search button.